extension/test/e2e/pageObjects/KlarnaPage.js

```diff
@@ -26,8 +25,15 @@ export default class KlarnaPage {
     await klarnaIframe.type('#otp_field', '123456')
 
     await klarnaIframe.waitForSelector(
-      '#invoice_kp-purchase-review-continue-button'
+      '#dd-confirmation-dialog__footer-button-wrapper'
+    )
+    const confirmDialog = await klarnaIframe.$(
+      '#dd-confirmation-dialog__footer-button-wrapper'
     )
-    await klarnaIframe.click('#invoice_kp-purchase-review-continue-button')
+
+    await klarnaIframe.evaluate((cb) => cb.click(), confirmDialog)
+    await klarnaIframe.waitForTimeout(2_000)
```

Comment:
Can't you waitForSelector rather than this?

Reply:
It is unnecessary, just for debug purpose in non-headless mode
https://github.com/commercetools/commercetools-adyen-integration/pull/972/commits/02a6a33ca22298c4374a9dd33ea23db5fb1e6c20